# HG changeset patch
# User Rik <rik@octave.org>
# Date 1511760041 28800
#      Sun Nov 26 21:20:41 2017 -0800
# Node ID 01fc0e70d4c16b61473e12d0cea175be4428f784
# Parent  0d4db91b72999d8e1c50af2143fd6682e00cdcc7
Use common coding style of 'htmp' passed to output 'h' for plot m-files.

* compass.m, feather.m, triplot.m: Use 'htmp' variable name for consistency
with other plot routines.

diff --git a/scripts/plot/draw/compass.m b/scripts/plot/draw/compass.m
--- a/scripts/plot/draw/compass.m
+++ b/scripts/plot/draw/compass.m
@@ -102,25 +102,25 @@ function h = compass (varargin)
   [r, p] = cart2pol (x, y);
 
   oldfig = [];
   if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
     hax = newplot (hax);
-    hlist = polar (r, p, line_spec);
+    htmp = polar (r, p, line_spec);
   unwind_protect_cleanup
     if (! isempty (oldfig))
       set (0, "currentfigure", oldfig);
     endif
   end_unwind_protect
 
   if (nargout > 0)
-    h = hlist;
+    h = htmp;
   endif
 
 endfunction
 
 
 %!demo
 %! clf;
 %! randn_9x1_data = [-2.555884; 0.394974; -0.191871; -1.147024; 1.355425; -0.437335; -0.014370; -0.941312; 1.240300];
diff --git a/scripts/plot/draw/feather.m b/scripts/plot/draw/feather.m
--- a/scripts/plot/draw/feather.m
+++ b/scripts/plot/draw/feather.m
@@ -101,25 +101,25 @@ function h = feather (varargin)
        ytmp - u * arrowsize / 3];
 
   oldfig = [];
   if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
     hax = newplot (hax);
-    hlist = plot (x, y, line_spec, [1, n], [0, 0], line_spec);
+    htmp = plot (x, y, line_spec, [1, n], [0, 0], line_spec);
   unwind_protect_cleanup
     if (! isempty (oldfig))
       set (0, "currentfigure", oldfig);
     endif
   end_unwind_protect
 
   if (nargout > 0)
-    h = hlist;
+    h = htmp;
   endif
 
 endfunction
 
 
 %!demo
 %! clf;
 %! phi = [0 : 15 : 360] * pi/180;
diff --git a/scripts/plot/draw/triplot.m b/scripts/plot/draw/triplot.m
--- a/scripts/plot/draw/triplot.m
+++ b/scripts/plot/draw/triplot.m
@@ -38,21 +38,21 @@
 function h = triplot (tri, x, y, varargin)
 
   if (nargin < 3)
     print_usage ();
   endif
 
   idx = tri(:, [1, 2, 3, 1]).';
   nt = rows (tri);
-  handle = plot ([x(idx); NaN(1, nt)](:),
-                 [y(idx); NaN(1, nt)](:), varargin{:});
+  htmp = plot ([x(idx); NaN(1, nt)](:),
+               [y(idx); NaN(1, nt)](:), varargin{:});
 
   if (nargout > 0)
-    h = handle;
+    h = htmp;
   endif
 
 endfunction
 
 
 %!demo
 %! clf;
 %! old_state = rand ("state");
